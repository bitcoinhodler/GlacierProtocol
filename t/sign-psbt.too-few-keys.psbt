cHNidP8BAHUCAAAAAUY2ibAZCqOl2mumdvUGp//w0ECaXSKWmPIrp6DoayXJAQAAAAD/////ApqhAAAAAAAAF6kUdi/qMkbcTOGc/T8aqWheiDhgUFeHoA8AAAAAAAAZdqkU+1tniKM2wM9JuQ5RrUJvLom/Q/uIrAAAAAAAAQDfAgAAAAH/6WtGqI1ibPOMhR5VlD9vBsc0eLsYAF2sP3q+aGkiBAEAAABqRzBEAiBq4GVVb99aCNdonNe6Lq7jtZrRRYKhg9JXIe7Njs6hhQIgeTPLaVDZ1w3qmedHvdmuD57WlTNzY5E+Hlx9W4l1JMUBIQMJ+zckzwn16IHam/KbrOYqHro9LC8Y7YKgGJMK9yyga/////8CMAz1BQAAAAAZdqkU/4wys506bAJs0AQeKKJtAC2h9gOIrFDDAAAAAAAAF6kUdi/qMkbcTOGc/T8aqWheiDhgUFeHAAAAAAEE/QsBUkEEd/FfIq7/rz87xIooCnZ/fGryEnZ4PAn/O8qrvs4XgRNc8PKPZD08YKdcKdsIGdVEK66D2PdOfETO0yVWZZmOUkEEA/ZCIjvHUeElq2BO0gxUTpM+uCoZIB+QvaYjap3b7ayTHBCMfDGJt/x+pwl7AiF9gZ6GaphBKRxSL2kXOhJSSkEEaJXz7QHAkCW6nPq3tn1pnj66zq2gvv9wvHLH8Q3y0cZ033Q7u1jOohFwgxM05NbZZe00ziMklVcJH0QDBaucAEEEnzyxUhMMJB4BuSIKFH9p9xEbgw3vNbUQlzq0e7mvf/byH8talYg5/uACLo6l3cbXH8YkqEvcQoNRk6v+4SWxw1SuAAEA/QsBUkEEd/FfIq7/rz87xIooCnZ/fGryEnZ4PAn/O8qrvs4XgRNc8PKPZD08YKdcKdsIGdVEK66D2PdOfETO0yVWZZmOUkEEA/ZCIjvHUeElq2BO0gxUTpM+uCoZIB+QvaYjap3b7ayTHBCMfDGJt/x+pwl7AiF9gZ6GaphBKRxSL2kXOhJSSkEEaJXz7QHAkCW6nPq3tn1pnj66zq2gvv9wvHLH8Q3y0cZ033Q7u1jOohFwgxM05NbZZe00ziMklVcJH0QDBaucAEEEnzyxUhMMJB4BuSIKFH9p9xEbgw3vNbUQlzq0e7mvf/byH8talYg5/uACLo6l3cbXH8YkqEvcQoNRk6v+4SWxw1SuAAA=
