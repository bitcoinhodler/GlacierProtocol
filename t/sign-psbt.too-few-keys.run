#!/bin/bash

# Not enough keys provided

$GLACIERSCRIPT --regtest=$1 sign-psbt << INPUT 2>&1
y
y
y
y
y
y
../../t/sign-psbt.too-few-keys.psbt
y
1
922F7weRdtBgCnqcnuMo8hba8wj75fZc2nx4oTnLHevh9GaCVXA
y
INPUT

retVal=$?
if [ $retVal -eq 0 ]; then
    echo "Error: expected glacierscript.py to fail"
    exit 1
fi
