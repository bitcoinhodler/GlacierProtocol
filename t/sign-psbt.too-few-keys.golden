Are you running this on a computer WITHOUT a network connection of any kind? (y/n)?Have the wireless cards in this computer been physically removed? (y/n)?Are you running on battery power? (y/n)?Are you running on an operating system booted from a USB drive? (y/n)?Is your screen hidden from view of windows, cameras, and other people? (y/n)?Are smartphones and all other nearby devices turned off and in a Faraday bag? (y/n)?Input a filename located in the current directory which contains the PSBT:
0.00050000 BTC in unspent inputs from cold storage address 2N42986nuqGfmbRUuojYCboE7EE9VLmYai2
0.00041370 BTC going back to cold storage address 2N42986nuqGfmbRUuojYCboE7EE9VLmYai2
0.00004000 BTC going to destination address n4S1Q6S9b5dq72ttpnFZXwaPCs2Z8ojwPR
Fee amount: 0.00004630
Confirm? (y/n): 
How many private keys will you be signing this transaction with (at least 2 required to complete transaction)?
#: ERROR: not enough private keys to complete transaction (need 2). Exiting...
